core/modules/ccontrol/MergingRequester.cc

```diff
@@ -227,11 +227,10 @@ bool MergingRequester::_merge() {
 }
 
 void MergingRequester::_setError(int code, std::string const& msg) {
-    LOGF_INFO("setError");
-    LOGF_INFO("setError %1% %2%" % code % msg);
+    LOGF_INFO("setError: code: %1%, message: %2%" % code % msg);
     std::lock_guard<std::mutex> lock(_errorMutex);
-    _error.code = code;
-    _error.msg = msg;
+    _error.setCode(code);
+    _error.setMsg(msg);
```

Comment:
`_error = Error(code, msg);` instead? Again, I do not like modifiable value-types, I'd prefer that setCode() method and other did not even exist in this class. And maybe `MergingRequester::_setError()` does not have to take two parametes but should take `Error` instead? But this is not for this ticket.